src/components/Timeline/Timeline.jsx

```diff
@@ -1,27 +1,23 @@
-import PropTypes from 'prop-types';
-import RecordItem from './RecordItem';
+import { useSelector } from 'react-redux';
 
-const Timeline = ({ passProcessEvents, validationBody }) => (
-  <ul className="px-2">
-    {passProcessEvents.map((processEvent, index) => (
-      <RecordItem
-        key={`validationEvent-${processEvent.createdAt}-${processEvent.user}`}
-        {...processEvent}
-        validationBody={validationBody}
-        first={index === 0}
-        last={index === passProcessEvents.length - 1}
-      />
-    ))}
-  </ul>
-);
+import RecordItem from './RecordItem';
 
-Timeline.propTypes = {
-  passProcessEvents: PropTypes.arrayOf(Object),
-  validationBody: PropTypes.string.isRequired,
-};
+const Timeline = () => {
+  const currentValidationRecord = useSelector((state) => state.currentValidationRecord);
```

Comment:
I don't think this should be here, I think the data flow should be the user opens a modal for a course validation record, and the modal which is the parent component makes the connection with the storage, his children just read the info sent by props.

Let me know what you think

Reply:
Thinking about the flexibility and reusability of the component, you are right. I'll be uploading the change!!